### ISSUE TITLE
Display of Cyrillic characters

### ISSUE THREAD

**Issue by Revoshell** [openmultiplayer/launcher#6]
It is necessary to add a normal display of Cyrillic characters. Many servers that use Cyrillic in their names have Cyrillic characters displayed as question marks.
By the way. In site monitoring, Cyrillic characters are also displayed crookedly.

**Comment by Awide** [comment]
imho not important

**Comment by Awide** [comment]
use latin letters not webdings

**Comment by Mortyyka** [comment]
We're talking about russians my guy

**Comment by AmyrAhmady** [comment]
> It is necessary to add a normal display of Cyrillic characters. Many servers that use Cyrillic in their names have Cyrillic characters displayed as question marks. By the way. In site monitoring, Cyrillic characters are also displayed crookedly.

Launcher is showing them correctly, website doesn't, which is source of launcher for getting information. 
Try adding that faulty server to favorites and see how it works and if it's still broken in launcher.

**Comment by Revoshell** [comment]
> > It is necessary to add a normal display of Cyrillic characters. Many servers that use Cyrillic in their names have Cyrillic characters displayed as question marks. By the way. In site monitoring, Cyrillic characters are also displayed crookedly.
> 
> Launcher is showing them correctly, website doesn't, which is source of launcher for getting information. Try adding that faulty server to favorites and see how it works and if it's still broken in launcher.

This is how Cyrillic characters are displayed everywhere.

**Comment by AmyrAhmady** [comment]
If possible, please send your discord username, so we can debug it on discord together @Revoshell 
or send me a message @iamir on discord

**Comment by AmyrAhmady** [comment]
Fixed in next release, thanks for your help. you can download it from nightly builds in here if you want to use a fixed version till new release comes out https://github.com/openmultiplayer/launcher/actions/runs/6833830624#artifacts
![image](https://github.com/openmultiplayer/launcher/assets/34688664/0a1dbd3f-a0a0-4b79-97d9-6d0b21ffddc7)

**Comment by Southclaws** [comment]
I went to check the API repo for the area of code which is responsible for this and realised it's missing!

There used to be a charset/codepage detection logic which worked pretty well for servers using old-school ANSI encoding. Ideally, we should all move to Unicode anyway and use that for Cyrillic. 

The library which I used to use for this is: https://github.com/gogs/chardet if anyone wishes to contribute this to the API that would be fantastic, and it would mean that Russian servers are rendered properly on the website as well as all other consumers of the API.

https://github.com/openmultiplayer/web
